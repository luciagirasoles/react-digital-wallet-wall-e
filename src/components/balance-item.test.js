import React from "react";
import { render } from "@testing-library/react";
import BalanceItem from "./balance-item";

test("balance-item component", () => {
  const category = { name: "Categoria 1", value: 34.6 };
  const { asFragment } = render(<BalanceItem category={category} />);
  expect(asFragment()).toMatchSnapshot();
});
