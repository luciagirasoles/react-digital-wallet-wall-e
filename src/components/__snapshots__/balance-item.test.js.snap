// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`balance-item component 1`] = `
<DocumentFragment>
  .emotion-1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 300px;
  margin: 2em 0;
}

.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  margin-left: 15px;
  font-size: 1.05em;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  height: 64px;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
}

<section
    class="emotion-1"
  >
    <img
      alt="Categoria 1"
    />
    <div
      class="emotion-0"
    >
      <span>
        Categoria 1 Category
      </span>
      <span>
        $ 34.6
      </span>
    </div>
  </section>
</DocumentFragment>
`;
